improver/blending/weighted_blend.py

```diff
@@ -475,10 +475,14 @@ def check_compatible_time_points(self, cube: Cube) -> None:
             ValueError : If blending over forecast reference time on a cube
                          with multiple times.
         """
-        if self.timeblending or cube.coord("time").ndim > 1:
+        if self.timeblending:
             return
 
-        time_points = cube.coord("time").points
+        try:
+            time_points = cube.coord("time_in_local_timezone").points
```

Comment:
I think that it would be good to add some mention of "time_in_local_timezone" to the docstring, just because we don't often use this coordinate.